tests/behat/features/bootstrap/WebFeatureContext.php

```diff
@@ -881,6 +887,26 @@ public function thereAreComments(TableNode $table)
     }
     $entity_manager->flush();
   }
+  /**
+   * @Given /^there are scratch remix relations:$/
+   *
+   * @throws Exception
+   */
+  public function thereAreScratchRemixRelations()
```

Comment:
for future tests, it would be better if the function is less hardcoded.
like defining the id and relation id.

E.g. take a look at thereArePrograms, ..